### PR TITLE
Mailsubscription -> UI/Confirmations: Make use of frontend_title if given

### PR DESCRIPTION
**Pull request by niels-heinemann**
Regardless of the profiles questions in https://github.com/civicrm/civicrm-core/pull/23325 we should use frontend_titles in our conversation with the subscribers in both UI notifications and confirmation emails.

### CODE REVIEW

File: CRM/Mailing/Event/BAO/MailingEventSubscribe.php [hunk 1]
```diff
@@ -341,12 +343,14 @@ public static function commonSubscribe($groups, $params, $contactId = NULL, $con
     $success = NULL;
     foreach ($groups as $groupID) {
       $title = CRM_Core_DAO::getFieldValue('CRM_Contact_DAO_Group', $groupID, 'title');
+      $fontend_title = CRM_Core_DAO::getFieldValue('CRM_Contact_DAO_Group', $groupID, 'fontend_title');
```
Comment (mattwire) [inline CRM/Mailing/Event/BAO/MailingEventSubscribe.php:346]: Should be `frontend_title`

Comment (niels-heinemann) [reply CRM/Mailing/Event/BAO/MailingEventSubscribe.php:346]: Jesses, yes!

**Comment by yashodha** [comment]
@niels-heinemann You might want to squash your commits.

**Comment by niels-heinemann** [comment]
Sorry, but what does that mean?

**Comment by agileware-justin** [comment]
@niels-heinemann what Yashodha is alluding too is that CiviCRM core team prefers that each PR have a single commit which includes all changes. The documentation is here, https://docs.civicrm.org/dev/en/latest/tools/git/#rebasing-interactive and also further above the page, https://docs.civicrm.org/dev/en/latest/tools/git/#committing

TBH the value of having someone actually take the time to submit a PR with a fix for a bug far outweighs the reply "squash your commits". Maybe that's just me :smile:

**Comment by niels-heinemann** [comment]
I tried for 20 min now - on github, on a local clone - no success. I'm crying. Maybe some day I have the time to open a new PR. Just close it and forget it.

**Comment by eileenmcnaughton** [comment]
@niels-heinemann I think we can switch to using the group tokens here `{group.frontend_title}` - but I'm looking at ALL the other discussions around frontend_title & it seems like we want to populate it on upgrade, make it required all over the place & have another mechanism to suppress. Anyway - just noting this & the related PR are on the radar as we work through the payment processor first

**Comment by niels-heinemann** [comment]
Thx a lot, @eileenmcnaughton! Looking forward to your advices. Maybe I can help.

**Comment by eileenmcnaughton** [comment]
I think we can replace this with https://github.com/civicrm/civicrm-core/pull/26913

**Comment by mattwire** [comment]
Replaced with #26913 (now merged).

**Pull request closed without merging**